src/main/java/com/devonfw/ide/sonarqube/common/impl/check/DevonArchitecture3rdPartyDatatypeMappingsCheck.java

```diff
@@ -0,0 +1,34 @@
+package com.devonfw.ide.sonarqube.common.impl.check;
+
+import org.sonar.check.Priority;
+import org.sonar.check.Rule;
+
+import com.devonfw.ide.sonarqube.common.api.JavaType;
+
+/**
+ * {@link DevonArchitecture3rdPartyCheck} verifying that datatypes are properly mapped
+ * @author lniazman
+ *
+ */
+@Rule(key = "Devon4j:E6", name = "Devon 3rd Party Datatype Mappings Check", //
+	description = "Verify that datatypes are mapped to the JPA using the correct annotation", //
```

Comment:
Sorry for being picky but can you write "using the correct pattern" instead of "annotation" as there is no correct annotation in the places this rule will identify.